submissions/basic_re/searcher.py

```diff
@@ -0,0 +1,7 @@
+import re
+
+str = "there is a needle in the haystack"
+
+x = re.findall(r'needle', str, re.IGNORECASE)[0]
+
+print("FOUND:", x)
```

Comment:
Add a trailing newline at the end of the file. see [here](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_392) if your interested in learning why